Romuashi/hometask-5/4-8.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM film cascade WHERE title = 'Gorko';
```

Comment:
this will not work if I select existing film
![Screenshot from 2021-11-23 21-08-31](https://user-images.githubusercontent.com/10963304/143080681-6f203950-29d8-4827-bfac-803d62bec2af.png)